apps/web/app/api/v1/client/responses/route.ts

```diff
@@ -27,10 +28,13 @@ export async function POST(request: Request): Promise<NextResponse> {
     );
   }
 
-  let survey;
+  let survey: TSurvey | null;
 
   try {
     survey = await getSurvey(responseInput.surveyId);
+    if (!survey) {
+      throw new Error("Survey not found");
```

Comment:
Can we instead of throwing return a ressource not found response to make the api clearer instead of throwing a 500 internal server error in the catch? :-)

Reply:
ahh nice idea, why did i miss it :(( nvm, pushed the change 💪🏼